components/maps/README.md

```diff
@@ -16,58 +18,67 @@ npm install @syncfusion/ej2-maps
+* [**Bubble**](https://ej2.syncfusion.com/angular/demos/#/material/maps/bubble?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Represents the underlying data values of the shapes.
+* [**Navigation line**](https://ej2.syncfusion.com/angular/demos/#/material/maps/navigationLines?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Denotes the path between two locations.
+* [**Color mapping**](https://ej2.syncfusion.com/angular/demos/#/material/maps/labels?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Applies colors to the shapes based on various conditions. Maps supports two types of color mapping: range color mapping and equal color mapping.
+* [**Legend**](https://ej2.syncfusion.com/angular/demos/#/material/maps/legend?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Denotes the color mapping categories.
+* [**Annotation**](https://ej2.syncfusion.com/angular/demos/#/material/maps/annotation?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Uses any custom HTML element as annotation and place it anywhere on the maps.
+* [**Selection and Highlight**](https://ej2.syncfusion.com/angular/demos/#/material/maps/selection?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Supports selecting or highlighting the shapes to bring center of attraction.
+* [**Zooming and Panning**](https://ej2.syncfusion.com/angular/demos/#/material/maps/zooming?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Allows to zoom in, out, and pan the map to show in-depth information.
+* [**Tooltip**](https://ej2.syncfusion.com/angular/demos/#/material/maps/tooltip?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Provides additonal information about the shapes on mouse hover.
+* [**Print and Export**](https://ej2.syncfusion.com/angular/demos/#/material/maps/print?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) - Provides options to print and export the rendered maps for future use.
 
 ## Support
 
 Product support is available for through following mediums.
 
-* Creating incident in Syncfusion [Direct-trac](https://www.syncfusion.com/support/directtrac/incidents?utm_source=npm&utm_campaign=maps) support system or [Community forum](https://www.syncfusion.com/forums/essential-js2?utm_source=npm&utm_campaign=maps).
-* New [GitHub issues](https://github.com/syncfusion/ej2-angular-ui-components/issues).
+* Creating incident in Syncfusion [Direct-trac](https://www.syncfusion.com/support/directtrac/incidents?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm) support system or [Community forum](https://www.syncfusion.com/forums/essential-js2?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm).
+* New [GitHub issues](https://github.com/syncfusion/ej2-angular-components/issues?utm_source=npm&utm_medium=listing&utm_campaign=angular-maps-npm).
```

Comment:
https://github.com/syncfusion/ej2-angular-components/issues - this link also broken. Please check.

Reply:
fixed